jib-core/src/main/java/com/google/cloud/tools/jib/registry/RegistryClient.java

```diff
@@ -246,14 +245,15 @@ public BlobDescriptor checkBlob(DescriptorDigest blobDigest)
    * written out.
    *
    * @param blobDigest the digest of the BLOB to download
-   * @param blobSizeConsumer callback to receive the total size of the BLOb to pull
-   * @param blobProgressListener listener for progress of the pull
+   * @param blobSizeListener callback to receive the total size of the BLOb to pull
+   * @param writtenByteCountListener listens on byte count written to an output stream during the
+   *     pull
    * @return a {@link Blob}
    */
   public Blob pullBlob(
```

Comment:
Maybe it could be simpler if these blob pull/push clsases took a single `ProgressEventDispatcherWrapper` with the set/dispatch methods instead of two consumers? I'm not sure how it fits into the class hierarchy though.